conf/singularity.config

```diff
@@ -0,0 +1,24 @@
+/*
+vim: syntax=groovy
+-*- mode: groovy;-*-
+ * -------------------------------------------------
+ *  Nextflow config file for use with Singularity
+ * -------------------------------------------------
+ * Defines basic usage limits and docker image id.
+ * Imported under the default 'singularity' Nextflow
+ * profile in nextflow.config
+ * NOTE: Not suitable for production use, assumes
+ * compute limits of only 16GB memory and 1 CPU core.
+ */
+
+singularity {
+  enabled = true
+}
+
+process {
+  container = { "docker://scilifelab/ngi-rnaseq:${workflow.revision ? workflow.revision : 'latest'}" }
```

Comment:
Why couldn't you use `wf_container` here?

Reply:
Should be no different to the [`uppmax` profile](https://github.com/SciLifeLab/NGI-RNAseq/blob/master/conf/uppmax.config#L19), no?

Reply:
No idea at all, when I tried "docker://$wf_container" all I got was: "docker://" without anything after...
Same with "docker://${wf_container}"...
Does that really work in the uppmax profile?

Reply:
We changed it recently, so it's possible that the change broke it. But it should do, yes. The docker profile used for automated testing uses it too: https://github.com/SciLifeLab/NGI-RNAseq/blob/master/conf/docker.config#L19

Reply:
I know that in the docker profile it works.
But it seems to me, that you can use the variable as it is, but you can't use it.
I don't really understand.
I'll try to make a reproducible minimal example to make an issue for Paolo

Reply:
Should be v. easy to run the uppmax test run script to check whether the uppmax config file actually works as expected.

Reply:
It is indeed very simple and it does currently not work:

Reply:
Ah, I bet we need to defer the execution of this line as well:

```groovy
container = { "docker://$wf_container" }
```

Reply:
Totally, but how do you intend to do that?

Reply:
With the snippet in my comment above I think.. Just use squiggly brackets around the entire expression.

Reply:
Oh I see...

Reply:
@ewels I tried what you suggested with no luck. Any other ideas?

Reply:
I'm not sure it's possible at all to do that right now with Nextflow.
I'll make an issue about it...